Looking at the failures:

```

2021-01-08 19:40:45.945074 \| ubuntu-bionic \| =========================== short test summary info ============================
--
1546 | 2021-01-08 19:40:45.945087 \| ubuntu-bionic \| FAILED test/integration/test_build.py::TestPytz::test_build_layer_reuse[podman]
1547 | 2021-01-08 19:40:45.945103 \| ubuntu-bionic \| ERROR test/integration/test_build.py::TestPytz::test_build_layer_reuse[docker]
1548 | 2021-01-08 19:40:45.945130 \| ubuntu-bionic \| ERROR test/integration/test_help.py::test_no_args[podman] - Failed: Running p...
1549 | 2021-01-08 19:40:45.945173 \| ubuntu-bionic \| ======== 1 failed, 37 passed, 5 skipped, 2 errors in 2903.67s (0:48:23) ========
```

These should get resolved by https://github.com/ansible/ansible-builder/pull/169

--- a/ansible_builder/constants.py
+++ b/ansible_builder/constants.py
@@ -12,7 +12,7 @@ base_roles_path = '/usr/share/ansible/roles'
 base_collections_path = '/usr/share/ansible/collections'
 
 build_arg_defaults = dict(
-    ANSIBLE_RUNNER_BASE_IMAGE='quay.io/ansible/ansible-runner:devel',
+    ANSIBLE_RUNNER_IMAGE='quay.io/ansible/ansible-runner:devel',
     PYTHON_BUILDER_IMAGE='quay.io/ansible/python-builder:latest'
 )
 

--- a/ansible_builder/main.py
+++ b/ansible_builder/main.py
@@ -317,14 +317,14 @@ class Containerfile:
         self.tag = tag
         # Build args all need to go at top of file to avoid errors
         self.steps = [
-            "ARG ANSIBLE_RUNNER_BASE_IMAGE={}".format(
-                self.definition.build_arg_defaults['ANSIBLE_RUNNER_BASE_IMAGE']
+            "ARG ANSIBLE_RUNNER_IMAGE={}".format(
+                self.definition.build_arg_defaults['ANSIBLE_RUNNER_IMAGE']
             ),
             "ARG PYTHON_BUILDER_IMAGE={}".format(
                 self.definition.build_arg_defaults['PYTHON_BUILDER_IMAGE']
             ),
             "",
-            "FROM $ANSIBLE_RUNNER_BASE_IMAGE as galaxy",
+            "FROM $ANSIBLE_RUNNER_IMAGE as galaxy",
             ""
         ]
 
@@ -425,7 +425,7 @@ class Containerfile:
     def prepare_final_stage_steps(self):
         self.steps.extend([
             "",
-            "FROM $ANSIBLE_RUNNER_BASE_IMAGE"
+            "FROM $ANSIBLE_RUNNER_IMAGE"
             "",
         ])
         return self.steps

--- a/docs/definition.rst
+++ b/docs/definition.rst
@@ -9,7 +9,7 @@ An example execution environment definition schema is as follows:
     version: 1
 
     build_arg_defaults:
-      ANSIBLE_RUNNER_BASE_IMAGE: 'quay.io/ansible/ansible-runner:stable-2.10.devel'
+      ANSIBLE_RUNNER_IMAGE: 'quay.io/ansible/ansible-runner:stable-2.10.devel'
 
     ansible_config: 'ansible.cfg'
 
@@ -36,7 +36,7 @@ to using the ``--build-arg`` CLI flag.
 
 Build args used by ansible-builder are the following.
 
-The ``ANSIBLE_RUNNER_BASE_IMAGE`` build arg specifies the parent image
+The ``ANSIBLE_RUNNER_IMAGE`` build arg specifies the parent image
 for the execution environment.
 
 The ``PYTHON_BUILDER_IMAGE`` build arg specifies the image used for

--- a/docs/usage.rst
+++ b/docs/usage.rst
@@ -87,7 +87,7 @@ directory. To specify another location:
 
 To use Podman or Docker's build-time variables, specify them the same way you would with ``podman build`` or ``docker build``.
 
-By default, the Containerfile / Dockerfile outputted by Ansible Builder contains a build argument ``ANSIBLE_RUNNER_BASE_IMAGE``, which can be useful for rebuilding Execution Environments without modifying any files.
+By default, the Containerfile / Dockerfile outputted by Ansible Builder contains a build argument ``ANSIBLE_RUNNER_IMAGE``, which can be useful for rebuilding Execution Environments without modifying any files.
 
 .. code::
 
@@ -97,7 +97,7 @@ To use a custom base image:
 
 .. code::
 
-   $ ansible-builder build --build-arg ANSIBLE_RUNNER_BASE_IMAGE=registry.example.com/another-ee
+   $ ansible-builder build --build-arg ANSIBLE_RUNNER_IMAGE=registry.example.com/another-ee
 
 
 ``--container-runtime``

--- a/test/data/build_args/base-image.yml
+++ b/test/data/build_args/base-image.yml
@@ -1,5 +1,5 @@
 ---
 additional_build_steps:
   prepend:
-    - ARG ANSIBLE_RUNNER_BASE_IMAGE
-    - RUN echo $ANSIBLE_RUNNER_BASE_IMAGE > /base_image
+    - ARG ANSIBLE_RUNNER_IMAGE
+    - RUN echo $ANSIBLE_RUNNER_IMAGE > /base_image

--- a/test/integration/test_build.py
+++ b/test/integration/test_build.py
@@ -157,10 +157,10 @@ def test_base_image_build_arg(cli, container_runtime, ee_tag, tmpdir, data_dir):
     ee_def = os.path.join(data_dir, 'build_args', 'base-image.yml')
     os.environ['FOO'] = 'secretsecret'
 
-    # Build with custom image tag, then use that as input to --build-arg ANSIBLE_RUNNER_BASE_IMAGE
+    # Build with custom image tag, then use that as input to --build-arg ANSIBLE_RUNNER_IMAGE
     cli(f'ansible-builder build -c {bc} -f {ee_def} -t {ee_tag}-custom --container-runtime {container_runtime} -v3')
     cli(f'ansible-builder build -c {bc} -f {ee_def} -t {ee_tag}-custom '
-        f'--container-runtime {container_runtime} --build-arg ANSIBLE_RUNNER_BASE_IMAGE={ee_tag}-custom -v3')
+        f'--container-runtime {container_runtime} --build-arg ANSIBLE_RUNNER_IMAGE={ee_tag}-custom -v3')
     result = cli(f"{container_runtime} run {ee_tag}-custom cat /base_image")
     assert f"{ee_tag}-custom" in result.stdout
 

--- a/test/test_cli.py
+++ b/test/test_cli.py
@@ -11,9 +11,9 @@ def test_custom_image(exec_env_definition_file, tmpdir):
     content = {'version': 1}
     path = str(exec_env_definition_file(content=content))
 
-    aee = prepare(['build', '-f', path, '--build-arg', 'ANSIBLE_RUNNER_BASE_IMAGE=my-custom-image', '-c', str(tmpdir)])
+    aee = prepare(['build', '-f', path, '--build-arg', 'ANSIBLE_RUNNER_IMAGE=my-custom-image', '-c', str(tmpdir)])
 
-    assert aee.build_args == {'ANSIBLE_RUNNER_BASE_IMAGE': 'my-custom-image'}
+    assert aee.build_args == {'ANSIBLE_RUNNER_IMAGE': 'my-custom-image'}
 
 
 def test_build_context(good_exec_env_definition_path, tmpdir):

--- a/test/test_main.py
+++ b/test/test_main.py
@@ -66,7 +66,7 @@ def test_base_image_via_build_args(exec_env_definition_file, tmpdir):
     assert 'ansible-runner' in content
 
     aee = AnsibleBuilder(
-        filename=path, build_args={'ANSIBLE_RUNNER_BASE_IMAGE': 'my-custom-image'},
+        filename=path, build_args={'ANSIBLE_RUNNER_IMAGE': 'my-custom-image'},
         build_context=tmpdir.mkdir('bc2')
     )
     aee.build()
@@ -74,14 +74,14 @@ def test_base_image_via_build_args(exec_env_definition_file, tmpdir):
     with open(aee.containerfile.path) as f:
         content = f.read()
 
-    assert 'ANSIBLE_RUNNER_BASE_IMAGE' in content  # TODO: should we make user value default?
+    assert 'ANSIBLE_RUNNER_IMAGE' in content  # TODO: should we make user value default?
 
 
 def test_base_image_via_definition_file_build_arg(exec_env_definition_file, tmpdir):
     content = {
         'version': 1,
         'build_arg_defaults': {
-            'ANSIBLE_RUNNER_BASE_IMAGE': 'my-other-custom-image'
+            'ANSIBLE_RUNNER_IMAGE': 'my-other-custom-image'
         }
     }
     path = exec_env_definition_file(content=content)
@@ -91,7 +91,7 @@ def test_base_image_via_definition_file_build_arg(exec_env_definition_file, tmpd
     with open(aee.containerfile.path) as f:
         content = f.read()
 
-    assert 'ANSIBLE_RUNNER_BASE_IMAGE=my-other-custom-image' in content
+    assert 'ANSIBLE_RUNNER_IMAGE=my-other-custom-image' in content
 
 
 def test_build_command(exec_env_definition_file):
@@ -170,8 +170,8 @@ class TestDefinitionErrors:
             "Keys ('middle',) are not allowed in 'additional_build_steps'."
         ),  # there are no "middle" build steps
         (
-            "{'version': 1, 'build_arg_defaults': {'ANSIBLE_RUNNER_BASE_IMAGE': ['foo']}}",
-            "Expected build_arg_defaults.ANSIBLE_RUNNER_BASE_IMAGE to be a string; Found a <class 'list'> instead."
+            "{'version': 1, 'build_arg_defaults': {'ANSIBLE_RUNNER_IMAGE': ['foo']}}",
+            "Expected build_arg_defaults.ANSIBLE_RUNNER_IMAGE to be a string; Found a <class 'list'> instead."
         ),  # image itself is wrong type
         (
             "{'version': 1, 'build_arg_defaults': {'BUILD_ARRRRRG': 'swashbuckler'}}",

